<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PT076LT" failures="1" tests="4" name="com.mycompany.carina.demo.WebSampleTest" time="189.124" errors="1" timestamp="2024-03-10T00:41:14 IST" skipped="0">
  <testcase classname="com.mycompany.carina.demo.WebSampleTest" name="testNewsSearch" time="41.643"/>
  <testcase classname="com.mycompany.carina.demo.WebSampleTest" name="testCompareModels" time="58.389">
    <failure message="The following asserts failed:
	expected [2023, February 29] but found [2023, February 01]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [2023, February 29] but found [2023, February 01]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at com.mycompany.carina.demo.WebSampleTest.testCompareModels(WebSampleTest.java:82)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
    <system-out>
      <![CDATA[TEST [Hello World test - testCompareModels] FAILED at [00:39:52 2024-03-10] - The following asserts failed:
expected [2023, February 29] but found [2023, February 01]

org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
com.mycompany.carina.demo.WebSampleTest.testCompareModels(WebSampleTest.java:82)
java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
java.base/java.lang.reflect.Method.invoke(Method.java:580)
org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
java.base/java.lang.Thread.run(Thread.java:1583)]]>
    </system-out>
  </testcase> <!-- testCompareModels -->
  <testcase classname="com.mycompany.carina.demo.WebSampleTest" name="testBrandGroup" time="54.103"/>
  <testcase classname="com.mycompany.carina.demo.WebSampleTest" name="testModelSpecs" time="34.989">
    <error message="Unable to open model: Galaxy A04" type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException: Unable to open model: Galaxy A04
at com.mycompany.carina.demo.gui.pages.desktop.BrandModelsPage.selectModel(BrandModelsPage.java:30)
at com.mycompany.carina.demo.WebSampleTest.testModelSpecs(WebSampleTest.java:47)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </error>
    <system-out>
      <![CDATA[TEST [Hello World test - testModelSpecs] FAILED at [00:40:30 2024-03-10] - Unable to open model: Galaxy A04

com.mycompany.carina.demo.gui.pages.desktop.BrandModelsPage.selectModel(BrandModelsPage.java:30)
com.mycompany.carina.demo.WebSampleTest.testModelSpecs(WebSampleTest.java:47)
java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
java.base/java.lang.reflect.Method.invoke(Method.java:580)
org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
java.base/java.lang.Thread.run(Thread.java:1583)]]>
    </system-out>
  </testcase> <!-- testModelSpecs -->
  <system-out>
    <![CDATA[TEST [Hello World test - testCompareModels] FAILED at [00:39:52 2024-03-10] - The following asserts failed:
expected [2023, February 29] but found [2023, February 01]

org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
com.mycompany.carina.demo.WebSampleTest.testCompareModels(WebSampleTest.java:82)
java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
java.base/java.lang.reflect.Method.invoke(Method.java:580)
org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
java.base/java.lang.Thread.run(Thread.java:1583)
TEST [Hello World test - testModelSpecs] FAILED at [00:40:30 2024-03-10] - Unable to open model: Galaxy A04

com.mycompany.carina.demo.gui.pages.desktop.BrandModelsPage.selectModel(BrandModelsPage.java:30)
com.mycompany.carina.demo.WebSampleTest.testModelSpecs(WebSampleTest.java:47)
java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
java.base/java.lang.reflect.Method.invoke(Method.java:580)
org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
java.base/java.lang.Thread.run(Thread.java:1583)]]>
  </system-out>
</testsuite> <!-- com.mycompany.carina.demo.WebSampleTest -->
